source/mainnet/net/voting/voting.rst

```diff
@@ -118,6 +120,20 @@ These older wallets do not have the capability to connect to dApps. But you do h
 
    #. It is possible to check from the Voting DApp that the delegation worked. The window for delegating votes is the same as voting with the DApp.
 
+.. dropdown:: Concordium Client
+
+    Users need to create a new account in |bw| or |cryptox| and delegate their vote from their Concordium Client wallet account(s) to the new account, then vote from the new account. You can create the new account in |bw| or |cryptox| at any time before the election.
+
+    **To delegate your vote:**
+
+    .. code-block:: console
+
+        $concordium-client transaction send --amount AMOUNT --receiver A --sender B --memo delegatevote2024
+
```

Comment:
In regards to the grpc that will be used, in agreement with SRE and Søren we should add disclaimer:
_There is no service license agreement for the grpc endpoint._

Reply:
It isn't clear to me exactly where I should put this disclaimer. I will integrated it in the numbered list item describing the concordium client command. If you prefer something else, please make another suggestion.

Reply:
It's fine. Thank you.